### PR TITLE
simple refactoring and performance improvements. no additional features

### PR DESCRIPTION
**Pull request by ilbonte**

### CODE REVIEW

**Comment by ibbatta** [comment]
Please @ilbonte fix and lint the code before

File: index.js [hunk 1]
```diff
@@ -70,26 +47,25 @@ io.on('connection', function(socket) {
...
+    }); 
   });
 }
 
 function saveMessageToDb(userId, message) {
   firebase.database().ref('/chat/messages/').push({
-    userId: userId,
-    message: message,
+    userId,
+    message,
```
Comment (ibbatta) [inline index.js]: @ilbonte could you explain me this short notation?

**Comment by ilbonte** [comment]
@ibbatta I don't know what's wrong with hound... do you have any idea?

**Comment by ibbatta** [comment]
@ilbonte honestly i don't know. I am investigating

**Pull request merged**